libs/wingsdk/src/cloud/topic.ts

```diff
@@ -77,11 +77,11 @@ export interface TopicSubscribeQueueOptions extends QueueProps {}
  */
 export interface ITopicClient {
   /**
-   * Publish message to topic
-   * @param message Payload to publish to Topic
+   * Publish messages to topic
+   * @param messages Payload to publish to Topic
```

Comment:
Please indicate in the docs that if multiple messages are passed then they will be published as a batch if supported by the target platform